docs/Getting Started.md

```diff
@@ -0,0 +1,113 @@
+
+  The command will also tag the image as `latest`.  This image tag should not be referenced in any Kubernetes YAML configuration, but serves only as a pointer, which is referenced when creating a Kubernetes stack.
+
+  The command aborts if any of the following conditions is met:
+  - The working directory is not clean (you have uncommit changes)
+  - The working directory contains any files specified in your `.gitignore` file
+  - The ECR project repository already contains the specified tag
+
+  The reason for these conditional checks is that when building, Docker will copy your _entire_ working directory into the container image, which can produce unexpected results when building images locally, destined for production environments!  The reason Hokusai aborts if it detects the working directory is unclean, or any ignored files or directories are present, is because it attempts to prevent any local configuration leaking into container images.
+
+  You can skip these conditional checks by supplying the `--force` flag.  Be careful of using `--force` when you have uncommited changes or ignored files in your working directory, especially when supplying `--force` without also supplying the `--tag` option.
+
+  Once an image is pushed, you can list the images and tags in the ECR project repository with: `hokusai images`.
+
+6) Create the Kubernetes staging environment
+  ```bash
+  hokusai env create --staging
+  hokusai env set RAILS_ENV=staging --staging
+  ```
+
+7) Create the Kubernetes staging stack with `hokusai stack create --staging`
+
+  `hokusai stack status --staging` should eventually (once Kubernetes creates a load balancer for your project), output the ELB's DNS record.
+
+  Get logs by running `hokusai logs --staging` and see deployment history with `hokusai history --staging`
+
+8) Create the Kubernetes production environment
+
+  Run the same command as above, but this time targeting the Kubernetes production environment:
+
+  ```bash
+  hokusai env create --production
+  hokusai env set RAILS_ENV=production --production
+  ```
+
+9) Create the Kubernetes production stack
+
+  Run the same command as above, but this time targeting the Kubernetes production environment:
+
+  ```bash
+  hokusai stack create --production
+  hokusai stack status --production
+  hokusai logs --production
+  hokusai history --production
+  ```
+
+10) Create a new commit and push it to the remote repo with `hokusai push` as before.
+
+11) Deploy the new commit to staging with `hokusai deploy {TAG} --staging` where TAG is the tag of the commit you just made.
+
+12) Promote the tag just deployed on staging to production
+  ```bash
+  hokusai promote
+  ```
```

Comment:
It doesn't need to go into this doc or PR necessarily, but it strikes me that we could use a reference of what a CI script should look like. Examples of what a typical deploy should look like to staging and then a promotion to production, basically.